payments-core/api/payments-core.api

```diff
@@ -901,7 +901,7 @@ public abstract interface class com/stripe/android/cards/CardAccountRangeReposit
 }
 
 public abstract interface class com/stripe/android/cards/CardAccountRangeService$AccountRangeResultListener {
```

Comment:
This class isn't actually public. We should probably add `@RestrictTo` the interface, and rerun apiDump to remove it entirely from this file.

Reply:
Thanks, updated!